app/notify_client/user_api_client.py

```diff
@@ -50,8 +50,8 @@ def get_user_by_uuid_or_email(self, user_uuid, email_address):
             "/user/get-login-gov-user",
             data={"login_uuid": user_uuid, "email": email_address},
         )
-        if user_data is None:
-            raise Exception("User not found")
+        if user_data is None or user_data.get("data") is None:
```

Comment:
What's the distinction between these two cases?  The user object straight up doesn't exist (`user_data` is `None`) or it does exist but is empty (`user_data` exists but the `data` attribute is `None`), or something else?

Reply:
excess of caution/defensive coding.  Or paranoia.  There's no difference.

Reply:
Fair enough, I have no qualms with double or triple checking possible failure points!